app/controllers/concerns/foreman/controller/bruteforce_protection.rb

```diff
@@ -10,6 +10,7 @@ def get_login_failures
   end
 
   def bruteforce_attempt?
+    return false unless session[:user].nil?
     limit = Setting[:failed_login_attempts_limit].to_i
     limit > 0 && get_login_failures >= limit
```

Comment:
perhaps moving the condition here would be slightly cleaner as it's the single definition of what is considered bruteforce attempt

`limit > 0 && get_login_failures >= limit && session[:user].nil?`